### DISCUSSION TITLE
tTIS simulation with simnibs

### DISCUSSION THREAD

**Discussion by BogdanI1011** [simnibs/simnibs#281] [2. General]
Is it possible to perform a simulation with tTIS? The simnibs GUI presents only tDCS as the alternative method, where just the current amplitude can be modified, not the frequency. Do you know can can this be adapted to tTIS or tACS?

**Comment by oulap** [comment]
Yes, but not in the GUI. Here are some references to get you started:

https://simnibs.github.io/simnibs/build/html/faq.html

https://github.com/simnibs/simnibs/blob/master/simnibs/examples/simulations/TI.py

https://www.sciencedirect.com/science/article/abs/pii/S1053811917307723

**Reply by octohhy** [reply]
Hi! The third link is down. Any other information? I want to perform a simulation with tACS.

**Reply by oulap** [reply]
I can access the third link, but here's a link to the preprint pdf for completeness:

https://backend.orbit.dtu.dk/ws/files/138455274/HKKR_1_s2.0_S1053811917307723_main.pdf

That describes how to incorporate the time component into the standard tDCS simulations.

**Reply by octohhy** [reply]
Ok, thx. ^_^

**Reply by octohhy** [reply]
Can I implement tACS simulation with simnibs if my frequency is 200Khz?

**Reply by Teddy0012** [reply]
My understanding: Strictly speaking, it is not feasible. Generally, the TI frequency range is 1K-10K, which can satisfy the quasi-static approximation, meaning it is no different from tDCS. For 200KHz, FEM should be solved in the frequency domain. However, I think you can give it a try because If you don't consider the impact of phase distribution, I think the intensity distribution at 200KHz should be similar to tDCS (no evidence, consider cautiously).
Could I ask a question why to use 200KHz？

**Reply by axelthi** [reply]
Hi, the quasi-static approximation and the assumption of tissues behaving as ohmic volume conductors is actually still valid at 200 kHz. See 
Predicting the electric field distribution in the brain for the treatment of glioblastoma
Pedro C Miranda, Abeye Mekonnen, Ricardo Salvador and Peter J Basser
 2014 Phys. Med. Biol. 59 4137 DOI 10.1088/0031-9155/59/15/4137
for a comparison.

**Reply by lingoodluck** [reply]
> My understanding: Strictly speaking, it is not feasible. Generally, the TI frequency range is 1K-10K, which can satisfy the quasi-static approximation, meaning it is no different from tDCS. For 200KHz, FEM should be solved in the frequency domain. However, I think you can give it a try because If you don't consider the impact of phase distribution, I think the intensity distribution at 200KHz should be similar to tDCS (no evidence, consider cautiously). Could I ask a question why to use 200KHz？

I have a question. The process set directly through the GUI should be the same as the actual experimental operation on the human body. The TI results I obtained directly using the GUI are different from the TI results achieved through the code you provided. In the code, you calculated TImax using a formula, but in the actual experiment, can I achieve the same effect by applying direct current (DC) electricity? I feel that the actual operation will be similar to the results from the GUI. In the actual operation, can I use direct current to achieve the same field distribution as TI?
